### PR TITLE
OWNxExplorer: fix reported/exported scene image 

### PR DESCRIPTION
**Pull request by kernc**
Fixes https://github.com/biolab/orange3/pull/1553#issuecomment-247312770

### CODE REVIEW

File: orangecontrib/network/widgets/graphview.py [hunk 1]
```diff
@@ -26,8 +26,7 @@ class QGraphicsEdge(QGraphicsLineItem):
     def __init__(self, source, dest, view=None):
         super().__init__()
         self.setAcceptedMouseButtons(Qt.NoButton)
-        self.setFlags(self.ItemIgnoresTransformations |
-                      self.ItemIgnoresParentOpacity)
+        self.setFlags(self.ItemIgnoresParentOpacity)
```
Comment (kernc) [inline orangecontrib/network/widgets/graphview.py]: Without `ItemIgnoresTransformations` the edges now scale with zoom, but, that was the only way I got it to work. 😊

**Comment by kernc** [comment]
Also, can someone please test wheelEvent zooming on Mac.

**Comment by ajdapretnar** [comment]
It looks ok, but oversized edges are somewhat of an issue. 🙈
I tried SaveImage and Report and both work fine. However, I can't seem to capture only a part of the network... For example, if I wish to observe only a small part of the network... I can't save the image or save it in the report. Something to consider.

WheelEvent seems a tad twitchy, but in essence it works.

**Comment by kernc** [comment]
> WheelEvent seems a tad twitchy, but in essence it works.

Worse than before?

**Comment by ajdapretnar** [comment]
I don't remember what *before* was, but it's not too tragic. Definitely passable.

**Comment by kernc** [comment]
Scaling edges are what I'm willing to settle with. Capturing the whole scene is consistent across widgets (cf. zoomed-inTree Viewer, Pythagorean Tree, ...).

If this works, it's ok by me.

**Comment by janezd** [comment]
No longer relevant after #83.

**Pull request closed without merging**